src/models/index.ts

```diff
@@ -0,0 +1,38 @@
+'use strict'
+
+const fs = require('fs')
+const path = require('path')
+const Sequelize = require('sequelize')
+const process = require('process')
+const basename = path.basename(__filename)
+const env = process.env.NODE_ENV || 'development'
+const config = require(__dirname + '/../../config/config.js')[env]
+const db: any = {}
+
+let sequelize: any
+if (config.use_env_variable) {
+  sequelize = new Sequelize(process.env[config.use_env_variable], config)
```

Comment:
It's better to put the db  connection in a separate file. Cause there may have ssl config, connection pool or debug, and other configs